src/components/ArticleList.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../utils/api";
+import ArticleCard from "./ArticleCard";
+
+function ArticleList() {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles()
+            .then(({ data: { articles } }) => {
+                setArticles(articles);
+            })
+            .catch((error) => {
+                console.log(error);
+            });
+    });
```

Comment:
What about if this request took a really long time to work? Some kind of loading state and message would go a long way for good user feedback. Inactive severs often take a while to spin up, so that loading state would give you confidence that the site was just waiting, and wasn't broken